### PR TITLE
Animation fixes and Spline Camera

### PR DESCRIPTION
**Pull request by GageM**
This branch has the character animation fixes as well as the spline-based camera

### CODE REVIEW

File: Source/unbread/Private/SCharacter.cpp [hunk 1]
```diff
@@ -120,8 +120,15 @@ void ASCharacter::Rotate(const FInputActionValue& Value)
 {
 	const FVector2D RotVector = Value.Get<FVector2D>();
 	const float Angle = FMath::Atan2(RotVector.Y, RotVector.X) * (180.0f / PI);
-	const FRotator NewRotation = FRotator(0.0f, -1* Angle, 0.0f);
-	GetMesh()->SetRelativeRotationExact(NewRotation);
+
+	// !!Safak please check if this change is okay with you!!
```
Comment (MashdorDev) [inline Source/unbread/Private/SCharacter.cpp:124]: @SafakOnol Check it out man

**Review by MashdorDev — COMMENTED** [review]
Looks great just need the attention of @SafakOnol in the SCharacter.cpp

**Comment by MashdorDev** [comment]
@GageM Don't forget to pull dev into the branch and push to the branch after you brought dev into your local branch

**Comment by GageM** [comment]
@MashdorDev Dev was pulled into the branch

**Comment by GageM** [comment]
That being said I can merge it again if you'd prefer

**Comment by GageM** [comment]
Alright, I just pulled Dev again and re-pushed

**Comment by MashdorDev** [comment]
Awesome, now we are just going to wait for @SafakOnol input on the code you asked him 👍

File: Source/unbread/Private/SCharacter.cpp (hunk 1 above)
Comment (SafakOnol) [reply Source/unbread/Private/SCharacter.cpp:124]: Approved!

**Review by SafakOnol — APPROVED** [review]
Good stuff

**Pull request merged**